docs/business.yaml

```diff
@@ -89,8 +89,8 @@ paths:
       summary: submit new filing
       operationId: post-businesses-identifier-filings
       responses:
-        '200':
-          description: OK
+        '201':
```

Comment:
Besides the 201 response, I am considering whether we should also add other responses like 401, 402, etc. 
My code only covers these two cases right now. Perhaps we could discuss other possible scenarios together and consider how to include corresponding schemas for these potential responses.

Reply:
You mean the other scenarios provided in the Postman collection? I personally don't think it's necessary, otherwise, there is no point to have the Postman collection. But we can double-check with @argush3 and Dave

Reply:
But you may be right. PPR did put the failed ones in it.

Reply:
added 3 pairs of failed examples with error messages 😄